src/test/java/jalau/at18/katas/likes/fernanda/MainTest2.java

```diff
@@ -0,0 +1,16 @@
+package jalau.at18.katas.likes.fernanda;
+
+import static org.junit.Assert.*;
+import org.junit.Test;
+
+public class MainTest2 {
+
+    @Test
+    public void shouldReturnHighestLowest() {
```

Comment:
You can add multiple tests in the same test class, no need to add new files if the tests are for the same class